plugins/backstage-plugin-coder/src/api/UrlSync.ts

```diff
@@ -48,7 +48,7 @@ type UrlPrefixes = Readonly<{
 
 export const defaultUrlPrefixes = {
   proxyPrefix: `/api/proxy`,
-  apiRoutePrefix: '/api/v2',
+  apiRoutePrefix: '', // Left as empty string because code assumes that CoderSdk will add /api/v2
```

Comment:
No, you're not missing anything. I guess I decided to keep the property because I thought that maybe down the line, it might make sense to let the prefix be dynamic

Even if we do need that later, though, it should be easy to add back later. I'll get rid of this